### PR TITLE
19808 Fixed vaults.gcp.env for entity-pay

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#19808

*Description of changes:*
I'm going to try testing after this PR is merged. Depending on what I find, i'll be making the code changes necessary.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-pay/devops/vaults.gcp.env [hunk 1]
```diff
@@ -1,12 +1,14 @@
+AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
+ENTITY_FILER_TOPIC="op://gcp-queue/$APP_ENV/payment/ENTITY_FILER_TOPIC"
+ENTITY_MAILER_TOPIC="op://gcp-queue/$APP_ENV/payment/ENTITY_MAILER_TOPIC"
```
Comment (argush3) [inline queue_services/entity-pay/devops/vaults.gcp.env:3]: I think this should be the following?  i.e. using the "topics" section

```
ENTITY_FILER_TOPIC="op://gcp-queue/$APP_ENV/topics/ENTITY_FILER_TOPIC"
ENTITY_MAILER_TOPIC="op://gcp-queue/$APP_ENV/topics/ENTITY_MAILER_TOPIC"
```

Comment (JazzarKarim) [reply queue_services/entity-pay/devops/vaults.gcp.env:3]: I see, so there's a vault for the topics. I thought each one would be in it's respective project. OK, I'll fix that too for the other vaults.gcp.env.

File: queue_services/entity-filer/devops/vaults.gcp.env [hunk 2]
```diff
@@ -32,7 +32,7 @@ NATS_FILER_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_FILER_SUBJECT"
 NATS_ENTITY_EVENT_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_ENTITY_EVENT_SUBJECT"
 NATS_EMAILER_SUBJECT="op://nats/$APP_ENV/nats-emailer/NATS_EMAILER_SUBJECT"
 AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
-TOPIC_NAME="op://gcp-queue/$APP_ENV/filer/TOPIC_NAME"
```
Comment (JazzarKarim) [inline queue_services/entity-filer/devops/vaults.gcp.env:35]: we can probably remove the TOPIC_NAME from the filer vault. Thoughts Argus?

Comment (argush3) [reply queue_services/entity-filer/devops/vaults.gcp.env:35]: Let's just keep the `TOPIC_NAME` for each service for now.  I think it was designed this way on purpose for the topic that is being subscribed to.

**Review by argush3 — APPROVED** [review]

**Pull request merged**